Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -152,4 +185,25 @@ extension WebView {
         /// The URL of the link that the user clicked.
         public let linkURL: URL?
     }
- }
+}
+
+extension WebView {
+    private enum Storage: DynamicProperty {
```

Comment:
And we don’t need to like forward `func update()` to the wrapped `State` here, right?